### PR TITLE
Issue 28

### PR DESCRIPTION
**Pull request by joneshf**
Ran into the same thing as here: https://github.com/pufuwozu/roy/issues/28

Objects can now have strings or numbers as their keys.

### CODE REVIEW

**Comment by joneshf** [comment]
This also affects #163.  I'm going to double check this later and probably merge it.

File: src/types.js [hunk 1]
```diff
@@ -220,7 +220,7 @@ ObjectType.prototype.toString = function() {
     for(p in this.props) {
         strs.push(p + ': ' + this.props[p].toString());
     }
-    return '{' + strs.join(', ') + '}';
+    return '{' + strs.join(', ').replace(/\'/g, '"') + '}';
```
Comment (puffnfresh) [inline src/types.js]: This is the only part I don't understand. What's this for?

**Comment by puffnfresh** [comment]
:+1:

File: src/types.js (hunk 1 above)
Comment (joneshf) [inline src/types.js]: It was in order to normalize the type output.  `{'a': 2}` would type as `{'a': Number}` whereas `{"a": 2}` would type as `{"a": Number}`.  Seemed a bit inconsistent to me as either version is a valid string and the repl would print the object as `{"a": 2}` in either case.  Also, it apparently didn't type nested quotes properly, so I've fixed that.

**Pull request merged**